src/components/swapv2/TokenInfoV2.tsx

```diff
@@ -1,12 +1,59 @@
 import React, { useEffect } from 'react'
-import { Currency } from '@kyberswap/ks-sdk-core'
+import { ChainId, Currency } from '@kyberswap/ks-sdk-core'
 import useTokenInfo, { TokenInfo } from 'hooks/useTokenInfo'
 import { useCurrencyConvertedToNative } from 'utils/dmm'
 import { TokenInfoWrapper } from './styleds'
-import SingleTokenInfo from 'components/swapv2/SingleTokenInfo'
+import SingleTokenInfo, { HowToSwap } from 'components/swapv2/SingleTokenInfo'
+import { TOKEN_INFO_DESCRIPTION, WHITE_LIST_TOKEN_INFO_PAIR } from 'constants/tokenLists/token-info'
+import { getSymbolSlug } from 'utils/string'
+import { useActiveWeb3React } from 'hooks'
 
 const isEmptyData = (tokenInfo: TokenInfo) => {
-  return !tokenInfo.price && !tokenInfo.description && !tokenInfo.tradingVolume && !tokenInfo.marketCapRank
+  return !tokenInfo.price && !tokenInfo?.description?.en && !tokenInfo.tradingVolume && !tokenInfo.marketCapRank
+}
+
+const checkTokenDescription = ({
+  tokenInfo1,
+  tokenInfo2,
+  tokenWrapped1,
+  tokenWrapped2,
+  chainId,
+}: {
+  tokenInfo1: TokenInfo
+  tokenInfo2: TokenInfo
+  tokenWrapped1: Currency | undefined
+  tokenWrapped2: Currency | undefined
+  chainId: ChainId | undefined
+}) => {
+  // hard code pair description for SEO
+  const rs1 = JSON.parse(JSON.stringify(tokenInfo1))
+  const rs2 = JSON.parse(JSON.stringify(tokenInfo2))
+  let isHardCode = false
+  if (tokenWrapped1 && tokenWrapped2) {
+    const mapByNetwork = WHITE_LIST_TOKEN_INFO_PAIR[chainId as ChainId]
```

Comment:
Dont make it chainid as ChainId, please find another way to handle undefined

Reply:
updated